docs/troubleshooting/cloud-functions-issues/_category_.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
both the files have similar titles, consolidate the information together and create a single doc or create multiple files if there are more than 1 topic with enough info in each.